MonoGame.Framework/Graphics/GraphicsMetrics.cs

```diff
@@ -9,54 +9,54 @@ namespace Microsoft.Xna.Framework.Graphics
     /// </summary>
     public struct GraphicsMetrics
     {
-        internal ulong _clearCount;
-        internal ulong _drawCount;
-        internal ulong _pixelShaderCount;
-        internal ulong _primitiveCount;
-        internal ulong _spriteCount;
-        internal ulong _targetCount;
-        internal ulong _textureCount;
-        internal ulong _vertexShaderCount;
+        internal long _clearCount;
+        internal long _drawCount;
+        internal long _pixelShaderCount;
+        internal long _primitiveCount;
+        internal long _spriteCount;
+        internal long _targetCount;
+        internal long _textureCount;
+        internal long _vertexShaderCount;
```

Comment:
Yeah, @KonajuGames suggested this change in https://github.com/MonoGame/MonoGame/issues/2433#issuecomment-110202391